justfile

```
@@ -69,7 +67,7 @@ EXEC_DEFAULTS := if IS_CI == "" { "" } else { "-T" }
 
 # Execute statement in service containers using Docker Compose
 exec +args:
```

I think we'll want to add `up` as a dependency for this, otherwise it'll fail if the stack isn't running:
```suggestion
exec +args: up
```

---

I was actually avoiding adding that dependency 😬 That's the reason why I don't use the `just dj` command here, so I don't have to see extra 10 lines in the terminal + the few seconds checking the container is already up every time I run something 😅

---

Ah that's fair 🙂 Definitely a bit of an annoyance waiting for the containers to start every time.